src/styles/main.scss

```
@@ -0,0 +1,4 @@
+@import "./utils/variables";
+@import "./utils/animations";
+@import "./utils/loops";
+@import "./blocks/clock.scss";
```

```suggestion
@import "./utils/variables";
@import "./utils/animations";
@import "./utils/loops";

@import "./blocks/clock.scss";
```

Blank line would be nice here to separate utils from blocks.